lsp-mode.el

```diff
@@ -4184,14 +3883,11 @@ in that particular folder."
 
 (defun lsp-configure-buffer ()
   (when lsp-auto-configure
+    (run-hooks 'lsp-configure-hook)
```

Comment:
Is this new hook? You forgot to define it?

Reply:
Yes a new hook, I Forgot to define it